tests/unit/test_response_util.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from flask import Flask, Response
+
+from app.versioning import LATEST_API_VERSION, VALID_API_VERSIONS
+from app.utils import standardize_response
+
+
+def test_presents_requested_api_version(app, client):
+    @app.route('/endpoint')
+    def endpoint():
+        return standardize_response({})
+
+    api_version = VALID_API_VERSIONS[0]
+    response: Response = client.get(
+        '/endpoint', headers=[('X-API-Version', api_version)])
+
+    assert response.json['apiVersion'] == float(api_version)
```

Comment:
The API version in the current response is a string, I'd like to keep it a string. I think this is where the confusion came from. If you go to https://resources.operationcode.org/api/v1/resources right now, you'll notice that it starts
```json
{"apiVersion":"1.0",
```
We can leave it as a string

Reply:
- [ ] change float conversion of API version in tests to keep it as a string